README.md

```diff
@@ -360,6 +361,11 @@ toxiproxy-cli toxic add -t latency -a latency=1000 shopify_test_redis_master
 
 Please consult your respective client library on usage.
 
+### 4. Logging
+
+There are the following log levels: panic, fatal, error, warn or warning, info, debug and trace.
+The level could be updated via environmental variable `LOG_LEVEL`.
```

Comment:
```suggestion
The level could be updated via environment variable `LOG_LEVEL`.
```

Nit: I understand the meaning, but "environment variable" has 10x more google hits and a more relevant first page - I think it's a better term for what you mean here.